streampipes-extensions/streampipes-processors-transformation-jvm/src/test/java/org/apache/streampipes/processors/transformation/jvm/Additional Tests

```diff
@@ -0,0 +1,96 @@
+import org.apache.streampipes.commons.exceptions.SpRuntimeException;
```

Comment:
This file misses our Apache license header, see an example [here](https://github.com/apache/streampipes/blob/ca4e0c2d44c1a713caaee3e9c1bff46096f41c53/streampipes-extensions/streampipes-processors-transformation-jvm/src/test/java/org/apache/streampipes/processors/transformation/jvm/processor/value/change/TestChangedValueDetectionProcessor.java#L1).